documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM1.adoc

```diff
@@ -0,0 +1,26 @@
+= Fuse Tooling - What's New in 10.3
+:page-layout: whatsnew
+:page-component_id: fusetools
+:page-component_version: 10.3.0.AM1
+:page-product_id: jbt_core
+:page-product_version: 4.5.3.AM1
+
+== New shortcuts in Fuse perspective
+
+Shortcuts for Java, Launch and Debug configuration and basic navigation are now enabled by default with Fuse perspective.
+
+The result is a set of actions now available in the Toolbar:
+
+image::./images/newToolbarAction.png[New Toolbar action]
+
+All the associated key shortcuts are also available, such as Ctrl+Shift+T to open a Java Type.
+
+== Performance improvement: loading Advanced tab for Camel components
```

Comment:
== Performance Improvements ==

Reply:
sure? there is a single performance improvement? need to put a 's'?

Reply:
It is a topic / subject only. actually who cares if there is one or ten improvements as long as there is a measurable improvement :D

Reply:
My suggestion is just to remove everything starting with the :

Reply:
as there is a single one, I find it more precise to specify it in the title